contents/docs/tutorials/1-minute/survey.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Running Surveys with No Backend
+sidebar: Docs
+showTitle: true
+---
+
+## Objective
+
+Running a user feedback survey without any backend.
+
+## Why is this useful?
+
+Getting user feedback is very important for improving your product. With PostHog, you run a survey without a backend, and can easily pair the survey data with all other metrics about your product.
+
+## Pre-Requisites
+
+To follow this tutorial along, you should:
+
+1. Have [deployed PostHog](/docs/deployment) or be using PostHog Cloud.
+
+> **Note:** PostHog surveys are best for binary or categorical data. If you want to collect arbitrary responses, PostHog might not be the most suitable tool.
+
+## Step-By-Step Instructions
+
+**1. Determine what you want to ask in your survey.**
+   
+Ideally, this data should be categorical, binary, interval-based, or on a scale. In other words, there should be a limited set of possibilities from which an answer is chosen, rather than accepting arbitrary data.
```

Comment:
We probably could collect this with text based feedback, but visualizing it is hard. Maybe it's own section. ie could it be viewed at a user level AND a hook or something used to alert you to the fact it has appeared?

Reply:
It would be pretty easy, just pass the answer through as text to the event, go to insights, break down by the property and you'll have a list of answers

Reply:
It's definitely easy, but I don't want to recommend it - it wouldn't be the nicest experience visualizing a bunch of paragraphs in PostHog. I can give more context though about my recommendations